### ISSUE TITLE
Updates to Comment Tab

### ISSUE THREAD

**Issue by BlakeDraper** [USGS-WiM/whispers#724]
Much this text taken directly from Neil in an email. 
- [x] 1. remove date last modified from comment details modal

- [x] 2. change "Location" column header to "Comment Source"

- [x] 3. for consistency with other areas of WHISPers, just use "Event" instead of "Event-wide" under Comment Source column (and it doesn't need to be italics)

- [ ] 4. ~the column sorting feature doesn't seem to work consistently... can you play around with this on the front end to see if you can make sense of it? Maybe some things are sorting by id number instead of alphabetically?~ 
Made into its own ticket: https://github.com/USGS-WiM/whispers/issues/729

- [x] 5. also, for created date sort, could we get it to sort by date and time (or record id?).  Often times, multiple comments will occur within the same day and would be good to be in the order for which they were created.

- [ ]  6. ~the "full" view displays funny on some computer screens, though works fine on others. There is a scroll bar (see below for scrolled left and scrolled right), but the background colors on the table aren't present in the two right-most columns~
made into it's own ticket: https://github.com/USGS-WiM/whispers/issues/730

- [ ] 7. ~service request comments don't seem to be being logged in the Comment Timeline.~
      Made into its own ticket: https://github.com/USGS-WiM/whispers/issues/727
- [ ] 8.~also, there is a need to show the service request RESPONSE in the comment pane.  This is obviously not a comment, but is important to see.  One potential solution is to auto create a comment upon response?  E.g. NWHC selects ‘Maybe’ from the response drop down and it creates an associated comment that reads “Response: Maybe”.  We’re open to suggestions here.~
     Made into its own ticket: https://github.com/USGS-WiM/whispers/issues/727

**Comment by lprivette** [comment]
Number 1-3 done in code. 

The are their own tickets now. 

@BlakeDraper For #5 in that list, there is no time data in the response, just the date e.g `2019-10-16`. Is this something we want to include in the response?

**Comment by BlakeDraper** [comment]
@lprivette  I failed to tell you this, but we actually put in a special field jut for sorting the comments by time. Its a special calculated field Aaron made juts for the `combined_comments` object for this very purpose. It's the `date_sort` field. That is what we should be sorting on. I should have either told you that or implemented it myself.

**Comment by lprivette** [comment]
Done in branch `727`

**Comment by JChipault** [comment]
Can you double-check the "time" sort? Yesterday, I entered test comments 1 (well, null), 2, and 3 (in that order) to event 170782 and today I entered test comments 4 and 5. I would expect to see 1, 2, 3, 4, 5 if sorted in one direction and 5, 4, 3, 2, 1 if sorted in the other direction but I see this:
![image](https://user-images.githubusercontent.com/49287076/67509212-1c11e300-f658-11e9-8b2c-17a8cd64bcab.png)

**Comment by JChipault** [comment]
I should add that I see the logic of what was coded, but in thinking about reading these comments like a story, I think having each day sorted independently could be hard to follow. There could be a lot of back and forth between people that would carry over from one day to the next. Is it easy to change the ascending sort, or complicated?

**Comment by lprivette** [comment]
I'll take a look and see if I can figure out what's wrong with the sort order.

For your question, if I'm understanding what you're asking then it should be easy. We can change the ascending sort to descending or none. Should I change it to none? This would mean that when the table loads it won't be sorted on 'Created Date'

**Comment by JChipault** [comment]
I think the default sort is great as is! (with most recent comment on the top). This is good:
![image](https://user-images.githubusercontent.com/49287076/67525110-a87fce00-f677-11e9-8a9e-da10e1d25dbe.png)

The trick is when you click the sort button on the "created date" column and attempt to sort with the oldest comment at the top, then if you tried to read the back-and-forth comments from the top to bottom and track the dialog over time it might not make much sense because they aren't sequential by time. the system seems to be putting the oldest day at the top but then within each day putting the newest comment at the top of that day. I'm looking for the oldest comment to be at the top and the newest comment to be at the bottom. So in the screenshot below, I was expecting to see test5 at the very bottom since that's the last comment I've entered into the system.
![image](https://user-images.githubusercontent.com/49287076/67525488-7458dd00-f678-11e9-824e-30ffd28af443.png)

**Comment by lprivette** [comment]
@JChipault Since there's a bunch of other done stuff in this ticket, I opened a new issue for this sorting problem here https://github.com/USGS-WiM/whispers/issues/731. I've got a fix for it working locally.
